upload-perf/src/upload-perf.rs

```diff
@@ -59,10 +59,13 @@ fn main() {
             if v["type"] == "bench" {
                 let name = v["name"].as_str().unwrap().trim_matches('\"').to_string();
 
-                last_commit = match get_last_metrics(&"commit".to_string(), &db, &name, &branch) {
-                    Result::Ok(v) => Some(v),
-                    Result::Err(_) => None,
-                };
+                if last_commit.is_none() {
```

Comment:
I think we can use `ok()` to avoid that boilerplate match
```rust
if last_commit.is_none() {
  last_commit = get_last_metrics(&"commit".to_string(), &db, &name, &branch).ok();
}
```